cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -209,6 +221,42 @@ func buildFlags(args *Args) []cli.Flag {
 			Usage:       "path to the kubeconfig file",
 			Destination: &args.KubeconfigPath,
 		},
+		&cli.BoolFlag{
```

Comment:
No worry about adding these startup flags, we will use a configuration file to configure nydus-snapshotter soon since it now has too many flags/options

Reply:
Maybe change this to `log-rotate-max-size`, `0` means disable rotation?